frontend/src/models/environment.ts

```diff
@@ -31,7 +31,7 @@ export class Environment {
         const url = this.accounts_url
         if (ENV.IS_DEV_MODE) return url
 
-        return `${url}/accounts/i/login/?r=${encodeURIComponent(window.location.href)}`
+        return `${url}/i/login/?r=${encodeURIComponent(window.location.href)}`
```

Comment:
You shouldn't need the /i for accounts anymore.

Reply:
oh good to know. thanks!